src/app/pages/MixerPage/MixerPage.tsx

```diff
@@ -1,32 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import type { MixerGear } from '../../../types';
 import Header from '../../components/Header/Header';
 import MixerCard from '../../components/MixerCard/MixerCard';
 import NavBar from '../../components/NavBar/NavBar';
 import style from './../MixerPage/MixerPage.module.css';
 
-const MixerCardMockData = [
-  { gearname: 'Moog Matriarch', channel: '19/20' },
-  { gearname: 'SH101', channel: '8' },
-  { gearname: '808', channel: '4' },
-  { gearname: 'Moog Matriarch', channel: '19/20' },
-  { gearname: 'SH101', channel: '8' },
-  { gearname: '808', channel: '4' },
-  { gearname: 'Moog Matriarch', channel: '19/20' },
-  { gearname: 'SH101', channel: '8' },
-  { gearname: '808', channel: '4' },
-  { gearname: 'Moog Matriarch', channel: '19/20' },
-  { gearname: 'SH101', channel: '8' },
-  { gearname: '808', channel: '4' },
-];
-
 export default function MixerPage(): JSX.Element {
+  const [gear, setGear] = useState<MixerGear[]>([]);
+
+  async function fetchGear() {
+    const response = await fetch('/api/gear', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    const gear = await response.json();
+    setGear(gear);
+  }
+
+  useEffect(() => {
+    fetchGear();
+  }, []);
```

Comment:
Again, can't you use `useFetch` here?

Reply:
Resolved in [d5b4171](https://github.com/Seborider/plugs./pull/48/commits/d5b417163f4008b402d101a241b31fbb985b8d6c)